creator/templates/ec_jwk.j2

```diff
@@ -0,0 +1,11 @@
+    {
+      "id": "{{ issuer }}#JWK2020-EC-key#{{ number }}",
+      "type": "JsonWebKey2020",
+      "controller": "{{ issuer }}",
+      "publicKeyJwk": {
+        "kty": "EC",
+        "crv": "{{ jwk.crv }}",
+        "x": "{{ jwk.x }}",
+        "y": "{{ jwk.y }}"
+      }
```

Comment:
This should be extended to optionally support `x5u` and `x5c` parameters as recommended by https://gaia-x.gitlab.io/technical-committee/identity-credentials-and-access-management-working-group/icam/credential_format/#json-web-key-2020

The same applies to the RSA-based template.